src/GitHub.Api/Application/ApiClient.cs

```diff
@@ -214,12 +211,12 @@ public ApiClient(UriString hostUrl, IKeychain keychain, IGitHubClient githubClie
                 var keychainAdapter = await keychain.Load(uriString);
 
                 var command = new StringBuilder("publish -r ");
-                command.Append(newRepository.Name);
+                command.Append(repositoryName);
 
-                if (!string.IsNullOrEmpty(newRepository.Description))
+                if (!string.IsNullOrEmpty(description))
                 {
                     command.Append(" -d ");
-                    command.Append(newRepository.Description);
+                    command.Append(description);
```

Comment:
This needs quotes around it for the command line to parse properly

Reply:
Addressed in #599